packages/driver/src/cypress/log.ts

```diff
@@ -348,10 +348,10 @@ export class Log {
       return this
     }
 
-    _.defaults(options, {
-      at: null,
-      next: null,
-    })
+    if (this.get('next')) {
```

Comment:
Any context on why we do this? I know this wasn't changed as part of this PR, but when I removed it and ran the tests, only navigation.cy.ts failed, specifically " it.only('logs before + after', () => {". The test doesn't exactly shine much light on the actual reason we do this, though.

Reply:
Right, it's the label for `before/after`. Without it, you get the id:

![image](https://user-images.githubusercontent.com/19196536/184780399-83b01f60-15ca-4c08-bfc8-dc74e1314579.png)

Reply:
Yep, just so. This was fairly arcane to me as well, escaped notice until the most recent commits.